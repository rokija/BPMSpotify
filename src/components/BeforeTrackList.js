import React from 'react';

const BeforeTrackList = () => {
    return (
        <div className="before-tracklist-wrapper">
            <div className="arrow-up glyphicon glyphicon-menu-up" />
            <span className="before-tracklist">Enter text in Search bar to get track details</span>
        </div>
    );
};

export default BeforeTrackList;
