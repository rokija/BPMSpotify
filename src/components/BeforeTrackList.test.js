import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import BeforeTrackList from './BeforeTrackList';

describe('BeforeTrackList', () => {
    const wrapper = shallow(<BeforeTrackList />);

    it('renders properly', () => {
        expect(wrapper.length).toEqual(1);
    });

    it('matches snapshot', () => {
        const tree = renderer.create(<BeforeTrackList />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
