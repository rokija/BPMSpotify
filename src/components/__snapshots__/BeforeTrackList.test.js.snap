// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeforeTrackList matches snapshot 1`] = `
<div
  className="before-tracklist-wrapper"
>
  <div
    className="arrow-up glyphicon glyphicon-menu-up"
  />
  <span
    className="before-tracklist"
  >
    Enter text in Search bar to get track details
  </span>
</div>
`;
